src/rustdoc_schema.graphql

```diff
@@ -1198,6 +1198,11 @@ interface FunctionLike {
   unsafe: Boolean!
   async: Boolean!
   has_body: Boolean!
+  """
+  Stringified representation of the function's signature, as close to the
+  original source code as possible.
+  """
+  signature: String!
```

Comment:
The description in the comment here seems to hint at deviations from the source code. Could you elaborate a bit on that? For example, it would be good to know if some property is never preserved, or if another one is always preserved, or maybe preserved and maybe not.

Reply:
It's the same limitations as the typename, I think - constants won't always be printed identically (esp. with constant expressions like `A<{1 + 2}>`), we can't recreate whitespace (but we try to pretty-print it), and I'm not sure if paths are guaranteed to be the same (but they will point to the same item)